Cargo.toml

```diff
@@ -17,17 +17,25 @@ travis-ci = { repository = "seladb/pickledb-rs" }
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-bincode = "1.0"
-serde_yaml = "0.8"
-serde_cbor = "0.11"
+
+serde_json = { version = "1.0", optional = true }
+bincode = { version = "1.0", optional = true }
+serde_yaml = { version = "0.8", optional = true }
+serde_cbor = { version = "0.11", optional = true }
 
 [dev-dependencies]
 rand = "0.6"
 rstest = "0.2"
 matches = "0.1"
 fs2 = "0.4"
 
+# 'bincode' feature created implicitly
```

Comment:
I'm not sure I understand this comment (maybe it's a lack of knowledge on my side). How come the `bincode` feature is created implicitly?

Reply:
When creating an optional dependency (`bincode`) the compiler automatically creates a feature for it, with the name of the dependency, that enables the dependency. Probably "automatically" is a better word than "implicitly" here, I was taking it from the Cargo book.
> By default, this optional dependency implicitly defines a feature

Reply:
Got it, thanks for the explanation! I'd suggest we add it anyway (although the compiler adds it automatically) just to make it easier to understand what are the possible features to turn on/off. What do you think?

Reply:
No problem, I'll put it in.